### DISCUSSION TITLE
Copy URL stream via wifi I2S( ESP32S3) to I2S (ESP32 Wrover Classic BT )  &  to  Bluetooth  speaker  via A2DP

### DISCUSSION THREAD

**Discussion by vednildip** [pschatzmann/arduino-audio-tools#1792] [Q&A] [answered]
Copy wifi URL stream from   I2S( decoding ESP32S3) to I2S (ESP32 Wrover Classic BT )  &  to  Bluetooth  speaker  via A2DP.  This is  too fast  to  process  URL data for  ESP32  Classic BT  (RX). How  to  throttle TX  at  ESP32S3 ?

**Comment by pschatzmann** [comment, marked as answer]
Check the format: A2dp needs 44100 samples per second, so you might need to resample
